build_runner/lib/src/entrypoint/options.dart

```diff
@@ -376,6 +375,26 @@ class _ServeCommand extends _WatchCommand {
   }
 }
 
+Future<HttpServer> _startServer(
+    _ServeOptions options, _ServeTarget target, ServeHandler handler) async {
+  var server = await _bindServer(options, target);
+  serveRequests(
+      server, handler.handlerFor(target.dir, logRequests: options.logRequests));
+  return server;
+}
+
+Future<HttpServer> _bindServer(_ServeOptions options, _ServeTarget target) {
+  switch (options.hostName) {
```

Comment:
@nex3 – we now do this 3-ish places, right?

pub, test, build_runner – could this be something we create as a util in `http_multi_server`?

Reply:
I'm not entirely sure what you mean by "do this"...